legend-engine-config/legend-engine-repl/legend-engine-repl-client/src/main/java/org/finos/legend/engine/repl/client/Client.java

```diff
@@ -70,12 +71,18 @@ public static void main(String[] args) throws Exception
     public MutableList<Command> commands;
 
     public Client(MutableList<ReplExtension> replExtensions, MutableList<CompleterExtension> completerExtensions, PlanExecutor planExecutor) throws Exception
+    {
+        this(replExtensions, completerExtensions, planExecutor, FileUtils.getUserDirectory().toPath());
```

Comment:
The default directory should not be the user home folder, but scoped to a folder like `.legend` or `.legend/repl` so the REPL somewhat mimick standard behavior of tools like `maven`, `npm`. We already persisted the command history and might potentially persist to disk a lot more things, so we should avoid overcrowding the user home dir.

```suggestion
        this(replExtensions, completerExtensions, planExecutor, FileUtils.getUserDirectory().toPath().resolve(".legend/repl"));
```

Reply:
Make sense!  Im makign that change.  Note that you dont need the `.`.

Reply:
the `.` will keep it as a hidden folder for `UNIX` environment, I think we should keep it.